connect.go

```diff
@@ -97,46 +100,85 @@ func (c *HCPConnectCommand) Run(args []string) int {
 }
 
 func (c *HCPConnectCommand) setupClients() error {
-	var opts []config.HCPConfigOption
-
-	if c.rmOrgClient == nil && c.rmProjClient == nil && c.vsClient == nil {
-		opts = []config.HCPConfigOption{config.FromEnv()}
-
-		if c.flagClientID != "" && c.flagSecretID == "" {
-			return errors.New("secret-id is required when client-id is provided")
-		} else if c.flagSecretID != "" && c.flagClientID == "" {
-			return errors.New("client-id is required when secret-id is provided")
-		} else if c.flagClientID != "" && c.flagSecretID != "" {
-			opts = append(opts, config.WithClientCredentials(c.flagClientID, c.flagSecretID))
-			opts = append(opts, config.WithoutBrowserLogin())
+	opts := []config.HCPConfigOption{config.FromEnv()}
+
+	if c.flagClientID != "" && c.flagSecretID == "" {
+		return errors.New("secret-id is required when client-id is provided")
+	} else if c.flagSecretID != "" && c.flagClientID == "" {
+		return errors.New("client-id is required when secret-id is provided")
+	} else if c.flagClientID != "" && c.flagSecretID != "" {
+		opts = append(opts, config.WithClientCredentials(c.flagClientID, c.flagSecretID))
+		opts = append(opts, config.WithoutBrowserLogin())
+	}
+
+	cfg, err := config.NewHCPConfig(opts...)
+	if err != nil {
+		return errors.New(fmt.Sprintf("Failed to connect to HCP: %s", err))
+	}
+
+	hcpHttpClient, err := httpclient.New(httpclient.Config{HCPConfig: cfg})
+	if err != nil {
+		return errors.New(fmt.Sprintf("Failed to connect to HCP: %s", err))
```

Comment:
Nit: `fmt.Errorf` would be better here

Reply:
There are a few `errors.New(fmt.Sprintf`s in this PR, but I'll just leave this comment once for both of our sanities :)

Reply:
I agree, will change. Thanks!